receiver/vcenterreceiver/metrics.go

```diff
@@ -103,7 +103,7 @@ func (v *vcenterMetricScraper) recordDatastoreProperties(
 	diskUsage := s.Capacity - s.FreeSpace
 	diskUtilization := float64(diskUsage) / float64(s.Capacity) * 100
 	v.mb.RecordVcenterDatastoreDiskUsageDataPoint(now, diskUsage, metadata.AttributeDiskStateUsed)
-	v.mb.RecordVcenterDatastoreDiskUsageDataPoint(now, s.Capacity, metadata.AttributeDiskStateUsed)
+	v.mb.RecordVcenterDatastoreDiskUsageDataPoint(now, s.Capacity, metadata.AttributeDiskStateAvailable)
```

Comment:
I agree the label was incorrect, but should the recorded value also be changed to `s.FreeSpace`? cc: @schmikei

Reply:
Ah yes you're right @BominRahmani we should be using `s.FreeSpace`